riscv-executor/src/lib.rs

```diff
@@ -127,7 +129,8 @@ instructions! {
     ec_add,
     ec_double,
     commit_public,
-    fail
+    fail,
+    keccakf
```

Comment:
I think this should not be uncommented here, otherwise we have to do manual witgen for keccak here too

Reply:
I assume you are referring to the other, and not to the instruction. In that case, what about the submachine macro? I'm partially guessing with the links and stuff, so any comments would be welcome :)

Reply:
Ah yes